pom.xml

```diff
@@ -16,6 +16,12 @@
   <description>Jenkins view that shows various cuts of build information via configured portlets.</description>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Dashboard+View</url>
 
+  <properties>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
```

Comment:
the project encoding was not specified, so the build is platform dependant.  (it would behave differently on a computer set to a Rusian locale vs an English locale.).
These three settings prevent most of the platform dependant, as the tools will pick up these settings rather than relying on the platform dependencies.